src/components/search/TourItem.jsx

```diff
@@ -1,15 +1,21 @@
+import { useMemo } from 'react'
 import { Link } from 'react-router-dom'
 import { Box, Typography } from '@mui/material'
 
 function TourItem({ attraction }) {
+  const city = useMemo(
+    () => (attraction?.address ? attraction.address.split(' ')[0] : ''),
+    [attraction],
+  )
```

Comment:
아주 사소한거지만 useMemo, useEffect는 아래와 같이 하는게 보편적이긴 합니다
```suggestion
  const city = useMemo(() => {
    return attraction?.address ? attraction.address.split(' ')[0] : ''
  }, [attraction])
```

Reply:
이런건 수정안하셔도 됩니당